package.json

```diff
@@ -17,14 +17,14 @@
   },
   "homepage": "https://bloxy.js.org/",
   "scripts": {
-    "test": "jest",
-    "test-with-coverage": "jest --coverage && cat ./coverage/lcov.info | codacy-coverage",
+    "test": "nyc mocha --require ts-node/register tests/**/*.spec.ts",
+    "test-with-coverage": "nyc report --reporter=text-lcov | codacy-coverage",
     "build": "tsc --skipLibCheck",
     "eslint": "eslint . --ext .ts",
-    "postinstall": "npm install got && echo \"\nThank you for installing bloxy! Dependency 'got' has been installed automatically, uninstall it if you plan to use bloxy in a browser-related project!\"",
     "prepublishOnly": "npm run build",
     "create-docs": "typedoc --plugin typedoc-plugin-markdown --hideBreadcrumbs --namedAnchors --excludeExternals --out docs src && mkdocs build",
-    "prepare-windows": "npm install && npm run build && npm run create-docs && echo 'Finished'"
+    "prepare-windows": "npm install && npm run build && npm run create-docs && echo 'Finished'",
+    "postinstall": "npm install got && echo 'The package got has been installed. You can now use the bloxy package.'"
```

Comment:
Since got is a standard dependency now, `npm install got` is not necessary anymore.
This is one of the things I removed on my fork too